### ISSUE TITLE
Not working with Intel XDK 5.1.1

### ISSUE THREAD

**Issue by FelipeCostaMauricio** [cranberrygame/cordova-plugin-ad-appodeal#1]
Hi! Im having trouble when adding  the plugin using this command "cordova-plugin-ad-appodeal", everytime I try to build it gets an error message. Im using the Cordova_Appodeal plugin version 1.0.6

**Comment by matrixreal** [comment]
got same problem here .... do you use construct 2 ?

**Comment by FelipeCostaMauricio** [comment]
Yes Construct 2, By the way Im Draconiz717 on scirra tutorials hehe, I noticed you posted questions there as well
